views/admin/data-contributions.html

```diff
@@ -5,27 +5,25 @@
 <table class="data-curation">
   <thead>
     <tr>
-      <th>Approved?</th>
       <th>User</th>
       <th>Trial</th>
       <th>File / URL</th>
       <th>Category</th>
       <th>Date</th>
-      <th class="comments-col">Comments</th>
       <th class="comments-col">Curator's comments</th>
     </tr>
   </thead>
   <tbody>
     {% for dataContribution in dataContributions %}
-    <tr>
-      <td>
-        {% if dataContribution.approved %}
-          Yes
-        {% else %}
-          No
-        {% endif %}
-      </td>
+      {% if dataContribution.approved %}
+        {% set status = "approve" %}
+      {% elif dataContribution.approved === false %}
+        {% set status = "reject" %}
+      {% else %}
+        {% set status = "unmoderated" %}
```

Comment:
As the class `unmoderated` isn't used for anything, you could simply set `status = ""`